### PR TITLE
Report issues 

### PR DESCRIPTION
**Pull request by yanyo87**
1. Fixes <https://github.com/demisto/etc/issues/36890>
update lodash & react-highlight packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "gulp": "3.9.1",
     "jquery": "3.5.0",
     "less": "2.7.2",
-    "lodash": "4.17.19",
+    "lodash": "^4.17.21",
```
Comment (orius123) [inline package.json:42]: Please use fixed version so we will know where we are at
```suggestion
    "lodash": "4.17.21",
```

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     "react": "16.8.5",
     "react-dom": "16.8.5",
     "react-grid-layout": "0.16.2",
-    "react-highlight": "briancappello/react-highlight#react-v16-compiled",
+    "react-highlight": "^0.14.0",
```
Comment (orius123) [inline package.json:61]:
```suggestion
    "react-highlight": "0.14.0",
```

**Review by orius123 — CHANGES_REQUESTED** [review]

**Comment by orius123** [comment]
Did you run a sanity with it?

**Comment by yanyo87** [comment]
> Did you run a sanity with it?

of course 

image

**Comment by orius123** [comment]
💯

**Review by Bars92 — APPROVED** [review]
👍

**Review by orius123 — APPROVED** [review]

**Pull request merged**